tests/tests.rs

```diff
@@ -21,7 +24,6 @@ async fn fairing_init() {
 #[rocket::async_test]
 async fn fairing_init_with_specific_traces_sampler() {
     let hub = Hub::current();
-    assert!(hub.client().is_none());
```

Comment:
Why remove these asserts?

Reply:
It was done in this comit: https://github.com/intgr/rocket-sentry/pull/80/commits/9cd4f67456e743c22f4d71a82ac7648da2cc29c0. Without it, it appears that since tests run in parallel, in the CI this test might have a Sentry's Hub already configured with a client.

We could also force all tests to run sequentially to prevent this error (`cargo test --jobs 1`). Since we don't have many tests it might be the easiest.

Reply:
The above comment is wrong `cargo test -- --test-threads=1` is the command to use. But it does make tests fail, because a previous test initialize the `Hub` and is kept between tests.

Reply:
I rollbacked the commit and forced the tests to run sequentially, but I didn't find a solution. Stopping the rocket server does not work and I didn't find any lead in [Sentry's own tests](https://github.com/getsentry/sentry-rust/tree/master/sentry/tests). Let me know if you have any ideas. Else I'm afraid we can't expect the Hub to be non-initialized at the start of a test.

Reply:
> We could also force all tests to run sequentially to prevent this error

Ah, no, I didn't realize it's testing global state. Let's just remove the assert for now, instead of introducing hacks to accommodate for it.

Maybe it would be an option to attach a custom `Hub` to `RocketSentry`, then we could check it in isolation. But that would be a separate PR.